ogre2/src/Ogre2SelectionBuffer.cc

```diff
@@ -222,18 +222,24 @@ void Ogre2SelectionBuffer::CreateRTTBuffer()
 
   Ogre::TextureGpuManager *textureMgr =
     ogreRoot->getRenderSystem()->getTextureGpuManager();
+  std::string selectionTextureName = "SelectionPassTex";
```

Comment:
name is different. is this wrong ? `"SelectionPassTex" + this->dataPtr->camera->getName(),` ?

Reply:
oh that's on purpose. The idea is to use the same texture for all ray queries.